index.html

```diff
@@ -20,24 +20,165 @@
+        <ul class="socials">
+          <li><a class="contacts" href="#"><img src="img/twitter-icon.png" alt='Twitter' /></a></li>
+          <li><a class="contacts" href="#"><img src="img/linkedin-icon.png" alt='linkedIn' /></a></li>
+          <li><a class="contacts" href="#"><img src="img/mail-icon.png" alt="Mail"/></a></li>
+          <li><a class="contacts" href="#"><img src="img/github-icon.png" alt="Github" /></a></li>
+          <li><a class="contacts" href="#"><img src="img/thumbup-icon.png" alt="Thumbup" /></a></li>
+        </ul>
+
+        <!-- Works Section -->
+         <section class="work">
+            <div class="grid-item frame-case-a">
+              <!-- <img src="img/Snapshoot-Portfolio-a.png" alt=""> -->
+              <div class="content">
+                <h3>Tonic</h3>
+                <ul class="flex">
+                  <li class="canopy">CANOPY</li>
+                  <li>Back End Dev</li>
+                  <li>2015</li>
+                </ul>
+              </div>
+
+              <!-- Work Section lists and write-up -->
+              <p>
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required.
+              </p>
+              <ul class="flex tool-content">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+              <button type="submit">See Project</button>
+            </div>
+
+            <!-- Multi-Post Stories Section  -->
+            <div class="grid-item frame-case-b">
+              <div class="content">
+                <h3>Multi-Post <br> Stories</h3>
+                <ul class="flex">
+                  <li class="canopy">CANOPY</li>
+                  <li>Back End Dev</li>
+                  <li>2015</li>
+                </ul>
+              </div>
+              <p>
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required.
+              </p>
+              <ul class="flex tool-content">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+              <button type="submit">See Project</button>
+            </div>
+            <div class="grid-item frame-case-c">
+              <div class="content">
+                <h3>Tonic</h3>
+                <ul class="flex">
+                  <li class="canopy">CANOPY</li>
+                  <li>Back End Dev</li>
+                  <li>2015</li>
+                </ul>
+              </div>
+              <p>
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required.
+              </p>
+              <ul class="flex tool-content">
+                <li>html</li>
+                <li>css</li>
+                <li>javascript</li>
+              </ul>
+              <button type="submit">See Project</button>
```

Comment:
Great job @dialite 🤝

- [ ] [Optional] As stated by the previous reviewer, kindly implement the button interactions of `hover`, and `active` across all buttons correctly.

Reply:
Thank you so much. I will act accordingly